waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java

```diff
@@ -192,28 +192,28 @@ public void shutdown() {
 
   //////////////////////////////
 
-  @Loggable(value = Loggable.DEBUG, skipResult = true, name = INVOCATION_LOG_NAME)
   @Override
+  @Loggable(value = Loggable.DEBUG, skipResult = true, name = INVOCATION_LOG_NAME)
```

Comment:
If the logger annotation is above the override one, checkstyle doesn't know it's an overriden method